### ISSUE TITLE
Confusions about Masked Operations.

### ISSUE THREAD

**Issue by zhuyr97** [haoyuc/MaskedDenoising#1]
Confusions about Masked Operations:

(1) Are masked operations necessary for both pixel and attention maps during both training and testing stages?

(2) According to Table 1 of the original paper, masked operations appear to be more important. What distinguishes the masked operations on attention maps from Attention Dropout operations? Will Attention Dropout achieve comparable performance?

**Comment by haoyuc** [comment]
We sincerely thank you for your attention and discussion of our work. Your insightful questions are truly appreciated and have contributed to a deeper understanding of our research.

(a) 
I don't think so, because the key component of the input mask can be applied to different network structures, including CNNs. Applying it to different architectures may require some modifications. 
However, we haven't conducted more experiments in this area yet. Applying this method to more scenarios is indeed one of our future research directions.

(b) 
There will be fluctuations, but the difference in quantitative values is minimal. We ran the model 50 times, and the PSNR standard deviation was only 0.00971. 
Moreover, Averaging multiple results bring a performance improvement, such as an increase of about 0.3dB in PSNR.

(c) 
During testing, we apply a mask to the attention features, which causes some information loss. Despite this, our model still performs well on noise types outside the training set. This supports our assertion that the standard training approach is overfiting the noise distribution in the training set. Therefore, when tested on a diverse and complex set of noise types outside the training set, some of the **network information used to overfit the training set distribution will be redundant**, and even if discarded, it will not cause significant performance loss. I believe this is very meaningful for the model's generalization performance and will be an interesting research direction.

**Comment by zhuyr97** [comment]
Thanks!
